annotation-model/TODO

```diff
@@ -0,0 +1,20 @@
+Things still to be developed:
+
+Script to run .jsonld files in tools/tests against .test files
+
+Script to batch run submitted .jsonld files from an implementation against .test
+files
+
+Websocket communication to runner tests with a corresponding listener that an
+implementation could use to auto-submit jsonld into the "manual" tests
+
+Change JSONtest logic so that it loads all of the required external files during
+initialization so that it can populate the list of assertions for the HTML in a
+manual test.  Use Promises and Promise.all() to achieve this.
+
+Add a Skip option to tools/runner
+
+Add a "or" mode for collections of assertions.
```

Comment:
The skip is something I want to add to runner - right now there is no way to say "I don't want to run this test right now."  

The or mode is supported in the comparisonObject handler.  I will update the TODO file.  Thanks!